### ISSUE TITLE
Filtering by tag support on dataplane view

### ISSUE THREAD

**Issue by johnharris85** [kumahq/kuma-gui#338]
### Description

We should add filtering support to the columns for all our tabular output, would make finding things in larger environments easier.

**Comment by jakubdyszkiewicz** [comment]
> Triage: we need more specific what we want to filter. We need to build better support on API side to have this in the GUI

**Comment by lahabana** [comment]
xref #382

**Comment by johncowen** [comment]
We spoke about this being a dupe of https://github.com/kumahq/kuma-gui/issues/464

We merged https://github.com/kumahq/kuma-gui/pull/469 which closed the above issue as we split out gateway type of proxies from standard types of proxies into separate listings - then the gateway listing uses the backend to filter between different types of gateway. Backend filtering is necessary otherwise you will only be filtering the data contained in the page of data you are on, not the total dataset on the backend.

So coming back around, I was unsure of closing this as a dupe as this says "filtering support for all columns" which is not complete, only filtering by dataplane proxy type (one column). I'm not totally sure if we have backend support for filtering by all the columns i.e. `Status`, `DP Version` etc.

TLDR; if someone else thinks this is complete, then feel free to close, otherwise I will need to figure out what filtering is available on the backend apart from `?gateway=`

**Comment by johncowen** [comment]
Ah gotcha, that makes sense then. So I think once this one is done (https://github.com/kumahq/kuma-gui/issues/337) along with https://github.com/kumahq/kuma-gui/issues/464 that is already closed, I guess that's when we can can close this one also, ty!

**Comment by lahabana** [comment]
Hmm not 100% sure. We should talk about this sync
